community/wyebadblock/depends

```diff
@@ -0,0 +1 @@
+webkit2gkt
```

Comment:
Typo. Should be `webkit2gtk` (You have `2gkt`).

Reply:
I have allready fixed that  and pushed a pr. Not there?